src/azure-cli-core/azure/cli/core/application.py

```diff
@@ -241,12 +241,22 @@ def _load_file(path):
             if path == '-':
                 content = sys.stdin.read()
             else:
-                try:
-                    with open(os.path.expanduser(path), 'r') as input_file:
-                        content = input_file.read()
-                except UnicodeDecodeError:
-                    with open(os.path.expanduser(path), 'rb') as input_file:
-                        content = input_file.read()
+                from codecs import open as codecs_open
```

Comment:
@derekbekoe, sure, I did a little bit more refactoring, so could you take another look?